plugins/kubernetes/app/controllers/kubernetes_clusters_controller.rb

```diff
@@ -0,0 +1,79 @@
+class KubernetesClustersController < ApplicationController
+  before_action :authorize_admin!
+  before_action :authorize_super_admin!, only: [ :create, :new, :update ]
+
+  before_action :find_cluster, only: [:show, :edit, :update]
+  before_action :load_environments, only: [:new, :edit, :update]
+  before_action :load_default_config_file, only: [:new, :edit, :update]
+
+  def new
+    @cluster = Kubernetes::Cluster.new(config_filepath: @config_file.try(:filepath))
```

Comment:
Actually, I shouldn't create the `ClientConfigFile` if the file doesn't exist. Will fix that logic below.